docs/reference/function-configuration/batching.md

```diff
@@ -0,0 +1,53 @@
+# Event batching
+
+(!) Feature is supported only for:
+* runtimes: [`python`]
+* trigger kinds: [`http`]
+
+The event batching feature allows passing a batch of events instead of a single event to a function handler. 
+The function invocation remains the same as in the usual flow. The processor keeps each invocation (or event) until 
+a batch is ready for sending. The batched event is sent to the runtime when either the batch is full or the batching
+timeout has passed. Both `batchSize` and `timeout` are configured per trigger.
+
+Example of trigger configuration with batching enabled:
+
+```yaml
+  triggers:
+    http:
+      class: ""
+      kind: http
+      name: http
+      batch:
+        mode: enable
+        batchSize: 10
+        timeout: 1s
+      maxWorkers: 10
+```
+
+A crucial difference from a single event flow lies in the necessity to manage responses for each event in a batch. 
+Every event in a batch has an `event_id` field, which should be used in the response when defining the corresponding response for each single event in a batched event. 
+By `response.event_id`, the processor will determine the correct target to send each single response. 
```

Comment:
The processor uses `response.event_id` to determine the correct target for sending each single response.